index.html

```diff
@@ -215,217 +588,409 @@ <h3>Code</h3>
   grid.load(items);
 &lt;/script&gt;
 </code>
-    </pre>
-          <p>
-            ...and that's it! Drag, drop, and resize to your heart's content.
-            Looking for more complex examples? Check out the advanced example below, the links in the nav
-            bar, or our docs in <a href="https://github.com/gridstack/gridstack.js/blob/develop/README.md" target="_blank">GitHub</a>
-          </p>
-        </div>
-      </div>
-    </section>
+</pre>
+        <p>
+          ...and that's it! Drag, drop, and resize to your heart's content.
+          Looking for more complex examples? Check out the advanced example
+          below, the links in the nav bar, or our docs in
+          <a
+            href="https://github.com/gridstack/gridstack.js/blob/develop/README.md"
+            target="_blank"
+            class="text-indigo-500 transition-colors hover:text-indigo-600"
+            >GitHub</a
+          >
+        </p>
+      </section>
 
-    <section id="advancedDemo" class="bg-primary">
-      <div class="row">
-        <div class="col-lg-12 text-white">
-          <ion-icon class="float-right" name="grid" style="font-size: 350%; margin-top: 10px"></ion-icon>
-          <h1>Advanced Demo</h1>
-          <hr />
-        </div>
-      </div>
-      <div class="row">
-        <div class="col-md-2 d-none d-md-block" style="border-right: 1px dashed white;">
-          <div id="trash" style="padding: 15px; margin-bottom: 15px;" class="text-center text-white">
-            <div>
-              <ion-icon name="trash" style="font-size: 400%"></ion-icon>
-            </div>
-            <div>
-              <span>Drop here to remove widget!</span>
-            </div>
+      <section id="advancedDemo" class="pt-40 mx-auto max-w-7xl">
+        <p
+          class="inline px-4 py-1 font-medium tracking-tight border border-gray-700 rounded-full"
+        >
+          Demonstration
+        </p>
+        <h2 class="mt-4 text-4xl font-medium">Advanced demo</h2>
+        <p>
+          A full control example with a trash can to remove widgets and a drag
+          in widget.
+          <br />
+          <br />
+          More complex examples are available to demonstrate the full power of
+          gridstack.js.
+          <br />
+          Drag and drop the items below to try!
+        </p>
+        <div class="grid grid-cols-2 gap-8 mt-8">
+          <div
+            class="flex flex-col items-center justify-center mx-auto transition-colors bg-green-100 rounded-lg cursor-pointer hover:bg-green-200 size-40 grid-stack-item-content grid-stack-item newWidget"
+          >
+            <ion-icon
+              name="add-circle"
+              class="text-4xl fill-green-700"
+            ></ion-icon>
+            <p class="text-center text-green-700">
+              Drag me into the dashboard!
```

Comment:
the style of the green item is causing it to not have margins when dropped into the grid. we shoould fix that.